compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -886,6 +907,13 @@ internal class RootNodeOwner(
                 postponed.clear()
             }
 
+            val isAnyCurrentFrameRateSet = !currentFrameRate.isNaN() || currentFrameRateCategory != 0f
+            if (isArrEnabled && isAnyCurrentFrameRateSet) {
+                platformContext.voteFrameRate(currentFrameRate, currentFrameRateCategory)
```

Comment:
Please add tests for this behaviour (inside `skikoTest`)

Reply:
Yes, still working on tests.